paasta_tools/kubernetes_tools.py

```diff
@@ -1602,6 +1603,7 @@ def get_kubernetes_metadata(self, git_sha: str) -> V1ObjectMeta:
                 "paasta.yelp.com/service": self.get_service(),
                 "paasta.yelp.com/instance": self.get_instance(),
                 "paasta.yelp.com/git_sha": git_sha,
+                "paasta.yelp.com/cluster": self.cluster,
```

Comment:
we probably wanna use `paasta_prefixed` here

Reply:
Will do!